src/vs/platform/workspace/common/workspaceTrust.ts

```diff
@@ -41,9 +41,10 @@ export interface IWorkspaceTrustManagementService {
 	onDidChangeTrustedFolders: Event<void>;
 
 	readonly workspaceResolved: Promise<void>;
+	readonly workspaceTrustEnabled: boolean;
 	readonly workspaceTrustInitialized: Promise<void>;
```

Comment:
silly thing, but this should be above the promises

Reply:
Done